app/src/resources/camera/usePreviewImage.ts

```diff
@@ -13,23 +16,44 @@ export interface UsePreviewImageResult {
 }
 
 export function usePreviewImage(
-  settings: CameraImageSettings
+  settings: CameraImageSettings,
+  runId: string | null
 ): UsePreviewImageResult {
   const [isLoading, setIsLoading] = useState(false)
-  const { data, refetch } = useCapturePreviewImage(settings, {
+
+  const previewQuery = useCapturePreviewImage(settings, {
```

Comment:
```suggestion
  const capturePreviewImage = useCapturePreviewImage(settings, {
```

See comments in `useAddCapturePreviewImageToRun.ts`